core/templates/pages/about-page/about-page.component.ts

```diff
@@ -23,13 +23,47 @@ import {SiteAnalyticsService} from 'services/site-analytics.service';
 import {UrlInterpolationService} from 'domain/utilities/url-interpolation.service';
 
 import './about-page.component.css';
+import {AccordionPanelData} from './data.model';
 
 @Component({
   selector: 'about-page',
   templateUrl: './about-page.component.html',
   styleUrls: ['./about-page.component.css'],
 })
 export class AboutPageComponent {
+  featuresData: AccordionPanelData[] = [
+    {
+      title: 'I18N_ABOUT_PAGE_FEATURE_TITLE1',
+      text: 'I18N_ABOUT_PAGE_FEATURE_SUBTEXT1',
+      customPanelClassNames: ['feature-panel'],
+      customTitleClassNames: ['feature-title', 'oppia-about-platform-subtext'],
+    },
+    {
+      title: 'I18N_ABOUT_PAGE_FEATURE_TITLE2',
+      text: 'I18N_ABOUT_PAGE_FEATURE_SUBTEXT2',
+      customPanelClassNames: ['feature-panel'],
+      customTitleClassNames: ['feature-title', 'oppia-about-platform-subtext'],
+    },
+    {
+      title: 'I18N_ABOUT_PAGE_FEATURE_TITLE3',
+      text: 'I18N_ABOUT_PAGE_FEATURE_SUBTEXT3',
+      customPanelClassNames: ['feature-panel'],
+      customTitleClassNames: ['feature-title', 'oppia-about-platform-subtext'],
+    },
+    {
+      title: 'I18N_ABOUT_PAGE_FEATURE_TITLE4',
+      text: 'I18N_ABOUT_PAGE_FEATURE_SUBTEXT4',
+      customPanelClassNames: ['feature-panel', 'free-of-cost-panel'],
+      customTitleClassNames: [
+        'feature-title',
+        'free-of-cost-title',
+        'oppia-about-platform-subtext',
+      ],
+    },
+  ];
+
+  panelIsCollapsed: boolean[] = [true, true, true, true];
```

Comment:
Instead of having this here separately, let's include this as part of the `featuresData` array items itself. This makes it more maintainable.

Reply:
I still see this array